### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by Aazam10**
I have created avatar and badge components for P0 requirements of component library please review

### CODE REVIEW

File: Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      <img
+        src="../images/avatar2.jpg"
+        alt="avatar"
+        class="avatar avatar-large"
+      />
+      <!-- Square-AVATAR -->
+
+      <img
+        src="../images/avatar2.jpg"
+        alt=""
```
Comment (VishalPatil18) [inline Avatar/avatar.html]: **Fix:** I would suggest giving value to the "alt" attribute as that helps for accessibility.

File: Avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar</title>
+    <link rel="stylesheet" href="../utility.css" />
+    <link rel="stylesheet" href="avatar.css" />
+  </head>
+  <body>
+    <div class="example-div">
+      <!-- PROFILE AVATAR -->
+      <img src="../images/avatar2.jpg" alt="avatar" class="avatar" />
```
Comment (VishalPatil18) [inline Avatar/avatar.html]: **Fix:** The images are located in a folder named "IMAGES", but here you've used "images" to refer to them. I think this should not work. Atleast it is not working in codesandbox. Just check this and rectify it everywhere.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
Apart from a couple of fixes, everything else looks good. Great Job Aazam 👍

File: Avatar/avatar.html (hunk 2 above)
Comment (Aazam10) [reply Avatar/avatar.html]: Hi vishal I have rectified the changes can you check once now

**Review by VishalPatil18 — APPROVED** [review]
Great it works fine now!

**Review by AtulPant2704 — APPROVED** [review]

**Review by starver20 — APPROVED** [review]

**Pull request merged**